src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -441,7 +443,23 @@ is_authorized(Req, State, Extractor) ->
             end;
         {false, ReqOther, StateOther} ->
             %% FIXME: the supported version is determined by the chef_authn application
-            %% also, see: https://wiki.corp.chef.io/display/CORP/RFC+Authentication+Version+Negotiation
```

Comment:
^ removed because this link is dead, and I can find no replacement for it in the new wiki.